### ISSUE TITLE
Use GitHub Web hooks

### ISSUE THREAD

**Issue by chriskacerguis** [ManageIQ/miq_bot#240]
It would be good to get the bot running on a platform where we can utilize the GitHub Web hooks vs. the "cron" interval that is running now.

**Comment by Fryguy** [comment]
Agreed, or possibly a mixture of both.  We may still need something like a polling style for times when the bot is down and doesn't receive the webhook, but once the bot has caught up, webhooks are much more efficient.

**Comment by chessbyte** [comment]
@chriskacerguis is there a first use case of this you had in mind?

**Comment by bdunne** [comment]
More discussion in #281
